eng/Version.Details.xml

```diff
@@ -9,10 +9,6 @@
 -->
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.AspNetCore.Components.WebAssembly.Runtime" Version="3.2.0">
```

Comment:
This package is gone. Happened to notice this when editing this file

Reply:
Please remove https://github.com/dotnet/aspnetcore/blob/f0a5ad6d68524860f40d305fd862dd86e6b551d3/eng/Versions.props#L135-L136 too

Reply:
Done